[CI] Builds fail because of upgrades for TypeScript.

```
tsconfig.json:4:25 - error TS5108: Option 'moduleResolution=node10' has been removed. Please remove it from your configuration.

4     "moduleResolution": "node",
                          ~~~~~~


Found 1 error in tsconfig.json:4

Build failed
```
It broke on July 8, five days ago, because "npm install -g typescript" in build.sh pulls in the latest stable release (as intended). In version 6.x, the compiler would just warn that it is deprecated--whatever that means. In version 7.0.2, M$ decided to remove the support completely.

--- a/_scripts/templates/TypeScript/tsconfig.json
+++ b/_scripts/templates/TypeScript/tsconfig.json
@@ -1,8 +1,7 @@
 {
   "compilerOptions": {
     "module": "ES2020",
-    "moduleResolution": "node",
-    "ignoreDeprecations": "6.0",
+    "moduleResolution": "bundler",
     "target": "ESNext",
     "noImplicitAny": true,
     "types": ["node"],

--- a/java/java/README.md
+++ b/java/java/README.md
@@ -88,4 +88,3 @@ literal, use [Trash trunfoldlit](https://github.com/kaby76/Trash/tree/main/src/t
 
 ## Reference
 * [pldb](http://pldb.info/concepts/java)
-

